rogue-thi-app/pages/personal.js

```diff
@@ -34,61 +68,151 @@ export default function Personal () {
           alert(e)
         }
       }
+
+      if (localStorage.debugUnlocked) {
+        setShowDebug(true)
+      }
     }
+
     load()
   }, [router])
 
-  /**
-   * Displays a row with the users information.
-   * @param {string} label Pretty row name
-   * @param {string} name Row name as returned by the backend
-   * @param {object} render Function returning the data to be displayed. If set, the `name` parameter will be ignored.
-   */
-  function renderPersonalEntry (label, name, render) {
-    return (
-      <ListGroup.Item>
-        {label}
-        <span className={userdata ? styles.personal_value : styles.personal_value_loading}>
-          <ReactPlaceholder type="text" rows={1} ready={userdata}>
-            {userdata && render && render()}
-            {userdata && !render && userdata[name]}
-          </ReactPlaceholder>
-        </span>
-      </ListGroup.Item>
-    )
-  }
-
-  return (
-    <AppContainer>
-      <AppNavbar title="Konto" />
-
-      <AppBody>
+  return (<AppContainer>
+    <AppNavbar title="Dein Profil"/>
+
+    <AppBody>
+      <ReactPlaceholder type="text" rows={20} ready={userdata}>
+
         <ListGroup>
-          {renderPersonalEntry('Matrikelnummer', 'mtknr')}
-          {renderPersonalEntry('Bibliotheksnummer', 'bibnr')}
-          {renderPersonalEntry('Druckguthaben', 'pcounter')}
-          {renderPersonalEntry('Studiengang', 'fachrich')}
-          {renderPersonalEntry('Fachsemester', 'stgru')}
-          {renderPersonalEntry('Prüfungsordnung', null, () => (
+          <ListGroup.Item action onClick={() => setShowPersonalDataModal(true)}>
+            <div className={styles.name_interaction_icon}>
+              <FontAwesomeIcon icon={faChevronRight} className="text-muted"/>
+            </div>
+            {userdata && userdata.name + ', ' + userdata.vname}<br/>
+            {userdata && userdata.fachrich}
+          </ListGroup.Item>
+
+          <ListGroup.Item className="text-muted">
+            <span className={userdata ? styles.personal_value : styles.personal_value_loading}>
+            {userdata && userdata.stgru + '. Semester'}<br/>
             <a
               /* see: https://github.com/neuland-ingolstadt/THI-App/issues/90#issuecomment-924768749 */
               href={userdata?.po_url && userdata.po_url.replace('verwaltung-und-stabsstellen', 'hochschulorganisation')}
               target="_blank"
               rel="noreferrer">
-              {userdata.pvers}
+            Deine SPO <FontAwesomeIcon icon={faExternalLink}/>
```

Comment:
Do we need the SPO in this location? I feel like this is a feature that would be very rarely used